index.bs

```diff
@@ -6576,11 +6560,22 @@ Constructors</h4>
 <dl dfn-type=constructor dfn-for="ConvolverNode/constructor()">
 	: <dfn>ConvolverNode(context, options)</dfn>
 	::
-		Let <var>node</var> be a new {{ConvolverNode}} object.
-		<a href="#audionode-constructor-init">Initialize</a>
-		<var>node</var>. Set an internal boolean slot
-		<dfn attribute for="ConvolverNode">[[buffer set]]</dfn>, and initialize it to <code>false</code>. Return
-		<var>node</var>.
+		When the constructor is called with a {{BaseAudioContext}} <var>c</var> and
+		an option object <var>option</var>, execute these steps:
+
+		1. Set the attributes {{ConvolverNode/normalize}} to the inverse of the
+			value of {{ConvolverOptions/disableNormalization}}.
+		2. If {{ConvolverNode/buffer}} is
+			<a href="https://heycam.github.io/webidl/#dfn-present">present</a>, set the
+			{{ConvolverNode/buffer}} attribute to its value.
+
+			Note: This means that the buffer will be normalized according to the
+			value of the {{ConvolverNode/normalize}}
+			attribute.
+
+		3. Let <var>o</var> be an empty dictionnary.
```

Comment:
We still want to process the `AudioNodeOptions` members on `options`.  One way to do that without the `ConvolverNode` special case in the "initialize" algorithm would be to let `o` be a new `AudioNodeOptions` dictionary with each dictionary member set to the corresponding value in `options`, if present.

Reply:
That's right, I totally forgot that this was something.